core/src/com/unciv/logic/city/CityStats.kt

```diff
@@ -433,7 +435,7 @@ class CityStats {
         if (cityInfo.getRuleset().modOptions.uniques.contains("Can convert gold to science with sliders")) {
             val amountConverted = (newFinalStatList.values.sumByDouble { it.gold.toDouble() }
                     * cityInfo.civInfo.tech.goldPercentConvertedToScience).toInt().toFloat()
-            if (amountConverted > 0) // Don't want you converting negative gold to negative science yaknow
```

Comment:
:(

Reply:
Didn't I just remove the "yaknow" which triggers the spellchecker? ... yup. Such language may be nice to lighten a mood - but aren't we in the worst possible mood beyond redemption when coding anyway? :wink: